css/style.css

```diff
@@ -9,6 +9,35 @@
 }
 
 /* BEGIN Header Menu */
+.menu-bar-container {
+  width: 100%;
+  height: 48px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  background: #3e3c3c;
+}
+
+.portfolio-menu {
+  display: none;
+  justify-content: flex-end;
+  margin-right: 10rem;
+  font-family: Inter, sans-serif;
+  font-size: 15px;
+  font-weight: 600;
+  line-height: 20px;
+  text-align: center;
+  align-items: center;
+}
+
+.desktop-menu {
+  display: none;
+}
+
+#hamburger {
+  cursor: pointer;
+}
```

Comment:
- [ ] As requested before, From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - Please make sure you remove the horizontal scroll you have, due to this horizontal overflow, after I click on the hamburger icon and want to close the mobile menu, I have to scroll horizontally before I will see the mobile menu close button.
  
[Screencast from 02-09-2023 12:15:31 AM.webm](https://user-images.githubusercontent.com/92303891/217682956-3dfca26e-6283-4985-bd03-c1b5b9196952.webm)
  
Please make sure your mobile menu design looks exactly like the Figma design